index.js

```diff
@@ -10,8 +10,8 @@ function redirect (url, rootSelector) {
   assert.equal(typeof url, 'string')
   assert.equal(typeof rootSelector, 'string')
 
-  return function redirectView (state, prev, send) {
-    send('location:set', url)
+  return function redirectView (state, emit) {
+    emit('pushState', url)
```

Comment:
This should become `replaceState` after https://github.com/yoshuawuyts/choo/pull/494 right?